infra/recipes/azure-sandbox/config/apps.tf

```diff
@@ -0,0 +1,62 @@
+resource "kubernetes_namespace" "sandbox" {
+  metadata {
+    name = "sandbox"
+  }
+}
+
+resource "kubernetes_deployment" "nginx_deployment" {
```

Comment:
We can clean up the nginx deployment part as that was added mainly for verifying services deployment using Terraform.